src/components/ToDoComponent/SettingGoals/SettingGoalsComponents/DeadLineDatePicker.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, StyleSheet, ScrollView } from 'react-native';
+import DatePicker from 'react-native-date-picker';
+
+const DeadLineDatePicker = () => {
+  const [date, setDate] = useState(new Date());
+  const [open, setOpen] = useState(false);
+
+  return (
+    <ScrollView>
+      <TouchableOpacity
+        style={styles.buttonStyle}
+        onPress={() => setOpen(true)}
+      >
+        <Text style={styles.buttonTextStyle}>Open calendar</Text>
```

Comment:
dont give this as text , it looks weird, you can just show the today's date by default and onPress you can show the calender.

Reply:
discussed !